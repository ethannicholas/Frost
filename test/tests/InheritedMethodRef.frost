method main() {
    Console.printLine(["A", "E", "G", "D", "C", "F", "B", "H"].sort(String.<))
}