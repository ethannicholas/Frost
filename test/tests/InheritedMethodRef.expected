[H, G, F, E, D, C, B, A]
